client/src/modules/Sidebar/components/Filter/FilterButton.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Badge from '@material-ui/core/Badge';
+import Tooltip from '@material-ui/core/Tooltip';
+import IconButton from '@material-ui/core/IconButton';
+import FilterListIcon from '@material-ui/icons/FilterList';
+
+const FilterButton = ({ isOpen, setIsOpen, filter }) => {
+  const toggleFilter = () => {
+    setIsOpen(prevState => !prevState);
+  };
+
+  return (
+    <IconButton
+      color="primary"
+      aria-label="filter"
+      onClick={toggleFilter}
+    >
+      {filter ? (
+        <Badge color="secondary" variant="dot">
+          <Tooltip
+            title={
+              isOpen
+                ? 'Close the filter'
+                : 'Open the filter'
+            }
+          >
+            <FilterListIcon />
+          </Tooltip>
+        </Badge>
+      ) : (
+        <Tooltip
+          title={
+            isOpen ? 'Close the filter' : 'Open the filter'
+          }
+        >
```

Comment:
Maybe create a const named tooltipTitle = isOpen ? 'Close the filter' : 'Open the filter'; and reuse her in both places.

Reply:
Fixed.